package.json

```diff
@@ -28,7 +28,8 @@
       "explorer": [
         {
           "id": "githubIssuesPrs",
-          "name": "GitHub Issues and PRs"
+          "name": "GitHub Issues and PRs",
+          "when": "is-running-a-github-repo"
```

Comment:
This appears to prevent the extension from activating with the `"onView:githubIssuesPrs"` event (see further up in this file). Activating on `"*"` should be good enough for now.